### DISCUSSION TITLE
Insteon Linking

### DISCUSSION THREAD

**Discussion by Boomer50BMG** [home-assistant/frontend#12743] [Other feature requests]
So far, I am using the Universal Devices ISY994 for my Insteon devices.  However, I would like to move them to Home Assistant with the Insteon integration.   However, the Device Linking capabilities on the ISY are quite a bit better.    I do not have to go to each device to link them.  I can do that in the interface.

Will that capability be added to the Insteon Integration in the near future?    If not, that is ok and I can keep the ISY994i around for the future.    I will be migrating all the Zwave functionality to Home Assistant and it is easier if everything is in one location.

**Comment by zsarnett** [comment]
https://github.com/teharris1/insteon-panel/issues

Please raise an issue with the Insteon Panel
